packages/flutter/lib/src/material/text_field.dart

```diff
@@ -604,6 +605,10 @@ class TextField extends StatefulWidget {
   /// [InputDecoration.enabled] property.
   final bool? enabled;
 
+  /// If false the text field is "disabled" but still scrollable.
+  /// However, if [enabled] is true, then the text field is "active" and still scrollable.
+  final bool? ignorePointer;
```

Comment:
Maybe instead of discussing the specific scrollable use-case this parameter is solving, you could talk about what it does in general. i.e. ignorePointer decides whether the widget will handle pointer events.

Reply:
+1, especially since this parameter doesn't actually cause the field to be disabled like `enabled` does.